packages/oid4vci-demo-frontend/src/configs/energy_shr.json

```diff
@@ -1,7 +1,7 @@
 {
   "general": {
     "credentialName": "EnergySHR Guest",
-    "issueCredentialType": "energySHR",
+    "issueCredentialType": "GuestCredential",
```

Comment:
why we're issuing GuestCredential all of a sudden?

Reply:
Huh that should be the type to begin with right. This is the Type that ends up in the credential and that should by GuesCredential for multiple envs like FMA, Triall, DBC

Reply:
This is correct, I'm just not sure why it's defined in two different places

Reply:
You mean the actionParam value? That was introduced later and probably this one was never removed, or still is being used by some code path.

Reply:
Reverted

Reply:
No it should not be reverted! Read please